tests/Unit/Symfony/OtelSdkBundle/Util/ExporterDsnParserTest.php

```diff
@@ -22,17 +24,17 @@ class ExporterDsnParserTest extends TestCase
         'password' => 'secret',
     ];
 
-    public function testParseFullDsn()
+    public function testParseFullDsn(): void
```

Comment:
are we missing a cs-fixer rule to enforce snake_case ?

Reply:
Yes, this has never been done in the contrib repo.